### ISSUE TITLE
Central thinning of the outer nuclear layer of the retina

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7337]
**Preferred term label:**
Central thinning of the outer nuclear layer of the retina

**Synonyms**
Central thinning of the ONL

**Definition (free text, please give PubMed ID)**
Reduced thickness of the outer nuclear layer in the middle region of the retina. The outer nuclear layer (ONL) of the retina contains the nuclei of the cone and rod photoreceptors. Loss of the cellular machinery found in these nuclei causes irreparable loss of the photoreceptors and the capacity for visual function.  This feature can be appreciated by directional optical coherence tomography
PMID:25829348

**Parent term (use hpo.jax.org/app)**
[Retinal thinning HP:0030329](https://hpo.jax.org/app/browse/term/HP:0030329)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
5/16
PMID:33891002
OMIM:619649

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000367
- label: Central thinning of the outer nuclear layer of the retina
- parents: [Retinal thinning (HP:0030329)](https://hpo.jax.org/app/browse/term/HP:0030329)
- pmid: PMID:25829348
- def: Reduced thickness of the outer nuclear layer in the middle region of the retina. The outer nuclear layer (ONL) of the retina contains the nuclei of the cone and rod photoreceptors. Loss of the cellular machinery found in these nuclei causes irreparable loss of the photoreceptors and the capacity for visual function. This feature can be appreciated by directional optical coherence tomography.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7337
- synonyms: Central thinning of the ONL(ABBREVIATION)
